request_handlers_test.cc

```diff
@@ -0,0 +1,111 @@
+#include "request_handlers.h"
+#include "gtest/gtest.h"
+
+#include <iostream>
+
+using namespace lightning_server;
+
+const int MAX_REQ_SIZE = 8192; // bytes
+
+class RequestHandlersTest : public ::testing::Test {
+protected:
+  // Helper method to pass arbitrary strings into the parser
```

Comment:
It looks like this comment was accidentally left-over from the nginx-configparser tests?